From 787ea1b835f6e780ee5fc9a9825970f460a221f3 Mon Sep 17 00:00:00 2001
From: Wallace Wang <r59996@freescale.com>
Date: Fri, 29 Aug 2008 20:18:49 +0800
Subject: [PATCH] ENGR00089719 MX35: Wrong sdma script address for ASRC

ASRC fails to convert the stream
The per_2_emi addr for ASRC use asrc_2_mcu, it should use shp_2_mcu

Signed-off-by: Wallace Wang <r59996@freescale.com>
---
 arch/arm/plat-mxc/sdma/sdma.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/plat-mxc/sdma/sdma.c b/arch/arm/plat-mxc/sdma/sdma.c
index 6da3017..819666a 100644
--- a/arch/arm/plat-mxc/sdma/sdma.c
+++ b/arch/arm/plat-mxc/sdma/sdma.c
@@ -344,7 +344,7 @@ static unsigned short sdma_get_pc(sdma_periphT peripheral_type,
 	} else if (peripheral_type == ASRC) {
 		switch (transfer_type) {
 		case per_2_emi:
-			res = sdma_script_addrs.mxc_sdma_asrc_2_mcu_addr;
+			res = sdma_script_addrs.mxc_sdma_shp_2_mcu_addr;
 			break;
 		case emi_2_per:
 			res = sdma_script_addrs.mxc_sdma_mcu_2_shp_addr;
-- 
1.5.4.4

